dev/release/rat_exclude_files.txt

```diff
@@ -13,6 +13,8 @@ ci/vcpkg/*.patch
 CHANGELOG.md
 cpp/CHANGELOG_PARQUET.md
 cpp/src/arrow/c/dlpack_abi.h
+cpp/src/arrow/flight/sql/odbc/odbc.def
+cpp/src/arrow/flight/sql/odbc/install/install_amd64.cmd
```

Comment:
Reminder to myself for writing in community PR:

The Apache Ignite [install_amd64.cmd](https://github.com/apache/ignite/blob/927ead13979937195aca6ffb717942fb477dc1b6/modules/platforms/cpp/odbc/install/install_amd64.cmd) file does not have a license on it, so I assume `.cmd` files don't need licenses. Same logic for `.def` files; the Apache Ignite [module.def](https://github.com/apache/ignite/blob/38a4804/modules/platforms/cpp/odbc/module.def) file does not have a license.